src/huggingface_hub/commands/lfs.py

```diff
@@ -119,43 +118,6 @@ def read_msg() -> Optional[Dict]:
     return msg
 
 
-class FileSlice(AbstractContextManager):
```

Comment:
Same here. If we're almost certain that people are not using this class in other libraries, we can remove it, but this might be breaking people's code. It'd be nice if @osanseviero and @LysandreJik could confirm this is not used in other places.

Reply:
Addressed in f419000

Reply:
> If we're almost certain that people are not using this class in other libraries

We are, it was just an internal obj for the LFS-specific part of the upload script. Very little chance that anyone ever used it. I would suggest to just remove